unit_tests/testAndersonAcceleration/testAndersonAcceleration.f90

```diff
@@ -60,539 +55,213 @@ PROGRAM testAndersonAcceleration
 !
 !-------------------------------------------------------------------------------
 SUBROUTINE testInit()
+
   CALL testAndAcc%init(mpiTestEnv,optList)
   ASSERT(testAndAcc%isInit,'%isInit')
```

Comment:
Need to see more of the `ASSERT_EQ` and similar updated macros throughout this test.

Reply:
No biggie. Can switch them over. Are there also ASSERT_LE and the like?